website/conferences/views.py

```diff
@@ -146,18 +146,21 @@ def _render_conference_node(node, idx):
         download_url = ''
         download_count = 0
 
-    author = node.visible_contributors[0]
+    if not downloads:
+        author = node.visible_contributors[0]
 
-    return {
-        'id': idx,
-        'title': node.title,
-        'nodeUrl': node.url,
-        'author': author.family_name,
-        'authorUrl': node.creator.url,
-        'category': 'talk' if 'talk' in node.system_tags else 'poster',
-        'download': download_count,
-        'downloadUrl': download_url,
-    }
+        return {
+            'id': idx,
+            'title': node.title,
+            'nodeUrl': node.url,
+            'author': author.family_name,
+            'authorUrl': node.creator.url,
+            'category': 'talk' if 'talk' in node.system_tags else 'poster',
+            'download': download_count,
+            'downloadUrl': download_url,
+        }
+    else:
```

Comment:
The `download` parameter and this conditional are unnecessary. The download count is already part of the return value in the dict.